src/main/java/org/hobbit/encryption/AES.java

```diff
@@ -0,0 +1,75 @@
+package org.hobbit.encryption;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.crypto.*;
+import javax.crypto.spec.SecretKeySpec;
+import java.security.InvalidKeyException;
+import java.security.NoSuchAlgorithmException;
+
+/**
+ * This class is a wrapper for Java AES encryption.
+ *
+ * @author Ivan Ermilov (iermilov@informatik.uni-leipzig.de)
+ *
+ */
+
```

Comment:
You mean that one instance of the class should not be shared between threads? My intended usage was to instantiate class every time encryption/decryption is necessary.

I can also push down instantiation of Cipher class to encrypt/decrypt methods if you think that can prevent issues in the future.

Reply:
1. Either create two ciphers (one for encryption and a second for decryption) and directly init them in the create method.
2. Simply add a comment to the javadoc that it is not thread safe. We don't need this class to be thread safe since you only use it in a single thread where it is created, used and destroyed afterwards.